README.md

```diff
@@ -111,4 +111,5 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md).
 
 <p align="center">Open sourced with ❤️ by <a href="https://noteable.io">Noteable</a> for the community.</p>
 
-[![boost data collaboration with notebooks](./assets/noteable.png)](https://noteable.io)
```

Comment:
I switched the link to point to our beta landing page (https://pages.noteable.io/private-beta-access) vs our noteable.io website.